src/client/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -31,6 +31,7 @@ public sealed partial class TokenCache : ITokenCacheInternal
     {
         internal const string NullPreferredUsernameDisplayLabel = "Missing from the token response";
         private const int DefaultExpirationBufferInMinutes = 5;
+        internal const int ExpirationTooLargeInDays = 10 * 365;
```

Comment:
>ExpirationTooLargeInDays [](start = 27, length = 24)

nit: is the expiration too large or too long? is the value here too big or the date too far off into the future?

Reply:
Too long, too long... :)